src/Struct/FindologicProduct.php

```diff
@@ -991,10 +991,19 @@ public function getCustomFields(): array
 
     /**
      * @param string|int|bool $value
-     * @return string|int
+     * @return array<string, int>|int|string
      */
     protected function getCleanedAttributeValue($value)
     {
+        if (is_array($value)) {
+            $values = [];
+            foreach ($value as $item) {
+                $values[] = $this->getCleanedAttributeValue($item);
+            }
+
+            return $values;
+        }
```

Comment:
Should handling arrays still be in this method? Or should this logic be put before the call `getCleanedAttributeValue` in `getCustomFieldProperties`?

Reply:
I was thinking the same and I also had both solutions implemented once. I went with this, as this is simple and also allows customers who use the extension plugin to call this method and get their data, cleaned.